app/experimenter/experiments/email.py

```diff
@@ -15,7 +15,7 @@ def send_review_email(experiment_name, experiment_url, needs_attention):
         ),
         attention=attention,
     )
-
+    assert settings.EMAIL_REVIEW
```

Comment:
Before, when the whole email sending function was mocked, the tests would depend on personal `.env` files. If you haven't set an `EMAIL_REVIEW` in *your* `.env` Django wouldn't bother sending any emails. 

For local dev, not that all emails are always only console printed, it'd be sad to NOT have an `EMAIL_REVIEW` set. For production, this is most definitely always set and won't ever change. And god forbid, it doesn't (incorrectly) gets unset it'd be nice to have an early error thrown here.

Reply:
If there is a problem, then this change would cause it to throw an error very late in the process. I think it would be better to leverage [Django's System Check framework](https://docs.djangoproject.com/en/2.1/topics/checks/) to check for the validity of these settings. That way it would prevent the server from ever starting in the first place.

Reply:
https://github.com/mozilla/experimenter/issues/950